lib/svgo/config.js

```diff
@@ -34,6 +34,7 @@ const pluginsOrder = [
   'collapseGroups',
   'convertPathData',
   'convertTransform',
+  'mergeStyles',
```

Comment:
I think it should be before minifyStyles an inlineStyles

Reply:
Done. Moved it above the two.